runtime/near-evm-runner/src/runner.rs

```diff
@@ -264,6 +264,7 @@ impl<'a> EvmContext<'a> {
             &self.fees_config.evm_config,
             self.chain_id,
         )?;
+        println!("{:?}", result);
```

Comment:
@ailisp Is this `println()` intentional?

Reply:
oops, will remove it